CHANGELOG.md

```diff
@@ -1,3 +1,50 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
I think it would be better to not have a table of contents in the `CHANGELOG.md`.

Reply:
I went ahead and removed it.